sys/pylon/gstpylonsrc.h

```diff
@@ -35,9 +35,17 @@ int gst_pylonsrc_ref_pylon_environment();
 int gst_pylonsrc_unref_pylon_environment();
 
 enum {
-  GST_PYLONSRC_NUM_CAPTURE_BUFFERS = 10
+  GST_PYLONSRC_NUM_CAPTURE_BUFFERS = 10,
+  GST_PYLONSRC_NUM_AUTO_FEATURES = 3,
+  GST_PYLONSRC_NUM_MANUAL_FEATURES = 2,
+  GST_PYLONSRC_NUM_LIMITED_FEATURES = 2
```

Comment:
Would these all be better off as just `#define`'s?

Reply:
I'm avoiding defines at all costs. Especially in headers. Though these cannot do much harm with such sophisticated names. Defines make compilation errors harder to understand as they do not include macro name. Enums are lightweight symbolic constants just like defines. They do not have memory address like const variables. But enums can only be used to represent integral constants

Reply:
Ok, I had not come across this approach before but makes sense.